lib/linguist/heuristics.rb

```diff
@@ -379,6 +379,14 @@ def call(data)
       end
     end
 
+    disambiguate ".proj" do |data|
+      if /^(\s*)(<Project|<Import|<Property|<?xml|xmlns)/i.match(data)
+        Language["XML"]
+      else
+        Language["Text"]
```

Comment:
This won't work because the `.proj` extension hasn't been registered as a filetype for the `Text` language. I believe heuristics are only for differentiating *shared* file extensions (used by more than one language type).

In other words, there's no need for this heuristic whatsoever.